solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/Settings/SettingsViewModel.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2021 Bending Spoons S.p.A. All rights reserved.
+
+import Foundation
+
+class SettingsViewModel {
```

Comment:
Given that we only have data, why don't we use a `struct` instead of a `class`? This will make the ViewModel leverage the value semantic and it will trigger the property observers automatically.